doc/examples/collision_environments/src/collision_scene_example.cpp

```diff
@@ -0,0 +1,100 @@
+#include <rclcpp/rclcpp.hpp>
+
+// MoveIt
+#include <moveit/common_planning_interface_objects/common_objects.h>
+#include <moveit/kinematic_constraints/utils.h>
+#include <moveit/move_group_interface/move_group_interface.h>
+#include <moveit/planning_scene/planning_scene.h>
+#include <moveit/planning_scene_interface/planning_scene_interface.h>
+#include <moveit/robot_model_loader/robot_model_loader.h>
+// TF2
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
+
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("collision_scene_example");
+
+int main(int argc, char** argv)
+{
+  rclcpp::init(argc, argv);
+  rclcpp::NodeOptions node_options;
+  node_options.automatically_declare_parameters_from_overrides(true);
+  auto collision_scene_example_node = rclcpp::Node::make_shared("planning_scene_tutorial", node_options);
+
+  // Check for robot_description_semantic param
+
+  moveit::planning_interface::PlanningSceneInterface planning_scene_interface_;
+  // Create vector to hold 3 collision objects.
+  std::vector<moveit_msgs::msg::CollisionObject> collision_objects;
+  collision_objects.resize(3);
+
+  // Add the first table where the cube will originally be kept.
+  collision_objects[0].id = "table1";
```

Comment:
This is fine for now, since you're just porting it and we know the vector has 3 elements already. But we usually prefer `.at()` over `[]`unless performance is very, very important. It's easier to debug